features/google/rollup.config.js

```diff
@@ -7,6 +7,7 @@ import commonjs from "@rollup/plugin-commonjs";
 import serve from "rollup-plugin-serve";
 import svg from "rollup-plugin-svg";
 import replace from "@rollup/plugin-replace";
+import sillyI18n from "../../feature-utils/silly-i18n/rollup-plugin.js";
```

Comment:
Having its own module (like another rollup plugin we have) would avoid this hard-coding, and substitute it by @polypoly-eu/pluginname

Reply:
`@polypoly-eu/silly-i18n` is already a package, so we can import the Rollup plugin as `@polypoly-eu/silly-i18n/rollup-plugin` without adding another package. This is consistent with `@polypoly-eu/podjs/rollup-plugin-gen-podjs/genPodjs.js`.

The only problem is that Rollup transpiles the configuration scripts and the resulting CommonJS code cannot import native ESM modules 🙄. So I had to convert the Rollup plugin to CommonJS which is not ideal.

Another solution would be to rename the various Rollup configuration scripts to `rollup.config.mjs` which causes Rollup to run it as native ESM (without transpiling the code), but then we need to touch our other Rollup plugins as well since they use `exports.default` which works different with Rollup's transpiler then it does in native Node.js. I think this owuld be a good idea though, but not as part of this change.

Reply:
You mean renaming *all* rollup files? Because we want to use a few to generate a file?

Reply:
No, it has nothing to do with generating modules. By default, Rollup transpiles the configuration script, essentially bundling `rollup.config.js` (and all imports referred by relative path) on the fly, while packaged modules are treated as external imports to be resolved by the Node.js environment. However, the code generated from the configuration file, is gonna be in CommonJS format and therefore cannot import ES modules. This worked so far because all our other Rollup plugins are not using ESM, and it will keep working if we use CommonJS for the Rollup plugin added here which is what I did for now.

Reply:
I'm sorry, but you totally lost me here. My initial point was related to putting rollup plugins in their own module. It does not really belong to `silly-i18n`, since there's no functional dependency with it.

Reply:
1. There is a dependency as the code generated by this plugin imports `@polypoly-eu/silly-i18n` and uses the `I18n` class.
2. `genPod.js` is provided in the same way (i.e. as part of the `@polypoly-eu/podjs` package). So the approach I chose here seems to be consistent with existing code.

Reply:
Well, still no functional dependency. There's nothing in the specific silly-i18n dependencies, or tests, or anything else, that's used by this file. Or the other way round.
Having it in its own module would be consistent with [`rollup-plugin-copy-watch`](https://github.com/polypoly-eu/polyPod/tree/main/dev-utils/rollup-plugin-copy-watch). Any reason why you've chosen to be consistent with that plugin, and not this one?

Reply:
I'm gonna have to cut this short, we're not gaining anything from this debate: I'm making the decision that the Rollup plugin goes into _silly-i18n_ (which was originally suggested by me, so no big surprise).